source/typescript.txt

```diff
@@ -0,0 +1,35 @@
+.. _typescript-center:
+
+==========
+TypeScript
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+Introduction
+------------
+
+Welcome to the documentation site for official MongoDB TypeScript support.
+Since Typescript builds on JavaScript, you can use the
+`Node.js Driver <https://www.mongodb.com/docs/drivers/node/current/>`__
+to create TypeScript applications that communicate with MongoDB.
+
+The Node.js Driver started supporting TypeScript with version 4.0. You must
+install that version of the driver or a more recent one to use TypeScript.
+
+To learn more about the features and limitations of using TypeScript with the
+Node.js Driver, see
+`Java Driver <https://www.mongodb.com/docs/drivers/node/current/fundamentals/typescript/>`__.
+
+Compatibility
+-------------
+
+For the compatibility charts for the Node.js Driver, see
+`Compatibility <https://www.mongodb.com/docs/drivers/node/current/compatibility/>`__
+in the driver docs.
```

Comment:
This is a tricky one. When I hear about compatibility w/r/t TypeScript, my mind jumps to a version of JS (or, in this case, the version of the Node.js driver that we've spent the whole page talking about). 

S: Remove the Compatibility section altogether. Open to talking this out, too!